Gemfile.lock

```diff
@@ -433,4 +433,4 @@ DEPENDENCIES
   yard
 
 BUNDLED WITH
-   2.4.14
+   2.3.24
```

Comment:
🤔 I don't like that this changed and on top of it actually regressed. I'll update in a dedicated commit.